constellations/gemini-admin/src/help/help.js

```diff
@@ -0,0 +1,33 @@
+"use strict";
+require("./help.styl");
+import React from "react";
+
+export class HelpPage extends React.Component {
+    render() {
+        return (
+            <div>
+                <h1>How to Create a Class</h1>
```

Comment:
since i'm working on unpublished/ published classes, you can either create another section for it or just include it in the details